service/controller/deleter_resource_set.go

```diff
@@ -6,26 +6,12 @@ import (
 	"github.com/giantswarm/operatorkit/v4/pkg/resource/wrapper/metricsresource"
 	"github.com/giantswarm/operatorkit/v4/pkg/resource/wrapper/retryresource"
 
-	"github.com/giantswarm/kvm-operator/service/controller/resource/cleanupendpointips"
 	"github.com/giantswarm/kvm-operator/service/controller/resource/node"
 )
 
 func newDeleterResources(config DeleterConfig) ([]resource.Interface, error) {
 	var err error
 
-	var cleanupendpointipsResource resource.Interface
```

Comment:
We don't need this resource any more because the new nodecontroller and the extra logic in the pod resource handler totally subsume what this did in a better way.